uo_sys_framework/ACREModule/cfgFunctions/common/fn_acreInit.sqf

```diff
@@ -211,9 +213,9 @@ if(!isDedicated) then
 						"UO_FW_ACRE_CIVILIAN_BABEL_CUSTOM","UO_FW_ACRE_CIVILIAN_BABEL_CUSTOM2"];
 
 			{
-				_index = _forEachIndex + 1;	
+				_index = _forEachIndex + 1;
```

Comment:
private _index
if you want to avoid an extra declaration one can also do (str (_forEachIndex + 1))